.travis.yml

```diff
@@ -1,8 +1,22 @@
-language: java
-
 matrix:
   include:
+    - name: deploy markbind
+      language: node
+      node_js:
+        - '8'
+      install:
+        - npm i -g markbind-cli
+      before_script: cd docs
+      script: markbind build
```

Comment:
can change this to 
```
script: cd docs && markbind build
```

Reply:
I felt it was clearer this way. But can.